dbux-projects/src/ProjectsManager.js

```diff
@@ -356,19 +355,20 @@ export default class ProjectsManager {
    * @param {Bug} bug 
    */
   async resetBug(bug) {
-    try {
-      await bug.project.gitResetHard(true, 'This will discard all your changes on this bug.');
+    const confirmMessage = 'This will discard all your changes on this bug. Are you sure?';
```

Comment:
Probably want to `deactivateBug()` first?

Reply:
use `ProjectManager.stopPractice` for consistency